packages/components/src/Subscription/Component.purs

```diff
@@ -159,11 +159,19 @@ render self@{ props: props@{ subscription: sub@{ package } } } =
             { wrapperState: self.state.wrapperProgress
             , readyView: pauseContainer [ pauseIcon, temporaryAddressChangeIcon, deliveryReclamationIcon ]
             , editingView: identity
-            , successView: pauseContainer [ DOM.div { className: "subscription--update-success check-icon" } ]
+            , successView: \succ -> case succ of
```

Comment:
As [JSX is a Monoid](https://pursuit.purescript.org/packages/purescript-react-basic/13.0.0/docs/React.Basic#t:JSX), we can use [foldMap](https://pursuit.purescript.org/packages/purescript-foldable-traversable/3.0.0/docs/Data.Foldable) here:

```purescript
successView: \msg -> pauseContainer [ foldMap successMessage msg, DOM.div { className: "subscription--update-success check-icon" } ]
```